projects/ui/src/shim.cds-core.scss

```diff
@@ -457,18 +457,32 @@
   --clr-alert-actions-spacing: var(--cds-global-space-6);
   --clr-alert-item-margin-bottom: var(--cds-global-space-2);
   --clr-alert-line-height: var(--cds-global-typography-font-size-7);
-  --clr-alert-sm-line-height: var(--cds-global-typography-font-size-6);
   --clr-alert-app-padding: var(--cds-global-space-4);
   --clr-alert-icon-wrapper-margin-top: 0; // ATM: delete after token migration
   --clr-alert-items-gap: var(--cds-global-space-5);
   --clr-alert-items-margin-right: 0; // ATM: delete after token migration
   --clr-alert-icon-margin-left: 0; // ATM: delete after token migration
   --clr-alert-close-icon-size: var(--cds-global-space-7);
+  --clr-alert-icon-size: var(--cds-global-space-9);
 
   .alert-app-level {
     --clr-alert-text-flex-grow: 0; // ATM: delete after token migration
   }
 
+  // Alert small
+  --clr-alert-sm-line-height: var(--clr-p2-line-height);
+  --clr-alert-sm-font-size: var(--cds-global-typography-font-size-3);
+  --clr-alert-sm-letter-spacing: var(--clr-p2-letter-spacing);
+  --clr-alert-sm-font-weight: var(--cds-global-typography-font-weight-regular);
+  --clr-alert-sm-icon-size: var(--cds-global-space-7);
+  --clr-alert-sm-icon-wrapper-height: var(--cds-global-space-7);
+  --clr-alert-sm-padding: var(--cds-global-space-3) var(--cds-global-space-6) var(--cds-global-space-3) var(--cds-global-space-5);
+  --clr-alert-sm-item-text-align-items: center;
+  --clr-alert-sm-close-icon-height: var(--cds-global-space-10); // ATM: delete after token migration
```

Comment:
This looks odd when selecting the button. The height/width of the button should be the same (24px), and we should add a token to increase the margin-top to get the correct vertical alignment.

Reply:
Is it ok like that?
![image](https://github.com/vmware-clarity/ng-clarity/assets/6692908/55431641-22d9-4bad-87c4-098172ebfaee)

This change is added in commit 3b37d84